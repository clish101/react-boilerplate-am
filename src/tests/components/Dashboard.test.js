import DashboardPage from '../../components/DashboardPage';
import { shallow } from 'enzyme';
import React from 'react';

test('Should show the dashboard page of the expenses', () => {
  const wrapper = shallow(<DashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
