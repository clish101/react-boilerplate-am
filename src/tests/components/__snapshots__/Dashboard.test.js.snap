// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should show the dashboard page of the expenses 1`] = `
<div>
  Dashboard page content
</div>
`;
